docs/en/cloud/Alluxio-Presto-AMI.md

```diff
@@ -0,0 +1,325 @@
+---
+layout: global
+title: Alluxio Presto Sandbox on AWS
+nickname: Alluxio Presto Sandbox
+group: Alluxio in the Cloud
+priority: 5
+---
+
+This guide describes how to launch a single node sandbox with Alluxio and Presto preconfigured to run queries.
+
+* Table of Contents
+{:toc}
+
+## Overview
+
+The Alluxio-Presto sandbox is an Amazon Machine Image offered on Amazon EC2
+and features installations of MySQL, Hadoop, Hive, Presto, and Alluxio.
+
+The goal of this guide is to show how Alluxio can improve Presto’s query performance
+by reading through Alluxio to access locally cached data, originally stored in an Amazon S3 bucket.
+
+## Prerequisites
+
+- An account on [AWS](https://aws.amazon.com/)
+- Web console access to [EC2](https://console.aws.amazon.com/ec2)
+- A terminal window with [SSH](https://man.openbsd.org/ssh.1)
+
+## Launch an EC2 instance with Presto + Alluxio stack
+{% accordion launch %}
+  {% collapsible Open the web console %}
+Open a web browser and navigate to the [EC2 console](https://console.aws.amazon.com/ec2), logging in with your credentials when prompted.
+Note the AWS region in the upper right corner.
+
+![presto_sandbox_ec2_region]({{ '/img/presto_sandbox_ec2_region.png' | relativize_url }})
+
+Click on the region to open a dropdown of available regions.
```

Comment:
region -> `region`?

Reply:
the button text is the name of the region, not the word `region`